test/conformance/ptp/ptp.go

```diff
@@ -123,6 +127,47 @@ var _ = Describe("[ptp]", func() {
 
 		})
 
+		Context("PTP Reboot discovery", func() {
+
+			BeforeEach(func() {
+				if fullConfig.Status == testconfig.DiscoveryFailureStatus {
+					Skip("Failed to find a valid ptp slave configuration")
+				}
+				ptpPods, err := client.Client.CoreV1().Pods(PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(ptpPods.Items)).To(BeNumerically(">", 0), "linuxptp-daemon is not deployed on cluster")
+
+				ptpSlaveRunningPods := []v1core.Pod{}
```

Comment:
Don't you already have this pod configured? Why would you want to rediscover this artifacts ?